### ISSUE TITLE
[Feature Request] Improved SSH Reconnect Experience

### ISSUE THREAD

**Issue by uponatime2019** [mRemoteNG/mRemoteNG#2651]
I frequently manage a large number of SSH connections (30+ VPS instances) using mRemoteNG and encounter frequent disconnects across various sessions. It would significantly improve usability and efficiency if the following features could be implemented to enhance the reconnection experience:

1. **Enhanced Disconnect Alert Navigation**  
   When an SSH connection is disconnected, an alert message currently appears. However, upon clicking "OK," it does not navigate to the relevant disconnected tab, requiring manual switching. I suggest an improvement where clicking "OK" in the disconnect message should automatically focus on and open the disconnected tab. This would streamline the workflow, especially when managing multiple tabs.

2. **Keyboard Shortcut for Reconnect**  
   Adding a keyboard shortcut to quickly reconnect a session would be very helpful. Currently, I have to manually click the tab and reconnect for each disconnected session. A shortcut (e.g., `Ctrl+Shift+R`) to trigger a reconnect action for the selected tab would greatly enhance productivity and make it easier to manage multiple sessions simultaneously.

Thank you for considering these suggestions!

**Comment by Kvarkas** [comment]
in NB its show disconect msg inside of console, noted the rest

**Comment by sovamind** [comment]
I fully support this request, but I wanted to add that it would be great if we could just have SSH integrated into the application. Having a separate instance of PuTTY installed isn't a great way to handle SSH for a console application that is supposed to be a single app for all consoles. I've got many places where they use PuTTY and it causes tons of confusion about which PuTTY they need to add saved sessions or which PuTTY agent they need to run for unlocking their keyfiles.

Seriously, just add the OpenSSH libraries to the application and let's be done with all this nonsense.

**Comment by simonai1254** [comment]
> I fully support this request, but I wanted to add that it would be great if we could just have SSH integrated into the application. Having a separate instance of PuTTY installed isn't a great way to handle SSH for a console application that is supposed to be a single app for all consoles. I've got many places where they use PuTTY and it causes tons of confusion about which PuTTY they need to add saved sessions or which PuTTY agent they need to run for unlocking their keyfiles.
> 
> Seriously, just add the OpenSSH libraries to the application and let's be done with all this nonsense.

I guess this is part of #181 and it's related issues (i.e. #1262).

**Comment by sovamind** [comment]
> I guess this is part of [#181](https://github.com/mRemoteNG/mRemoteNG/issues/181) and it's related issues (i.e. [#1262](https://github.com/mRemoteNG/mRemoteNG/issues/1262)).

Yeah, that would solve the problem too. I think I also saw someplace that they are replacing the connections file with a more secure database that will be able to store the keyfiles and passwords instead of the flat file they are using now.  Guess we just need to wait for all the goodies.

**Comment by robertpopa22** [comment]
Triage refresh (P4 version-label debt): this issue is tagged with legacy version labels. Please retest on the latest build (fork v1.79.x or upstream dev), then share reproducible steps and expected vs actual behavior. If still valid, keep open and relabel to current triage scope (for example Need 2 check). If fixed, close with resolution.

Current legacy version labels: 1.77.4
